class/defaults.yml

```diff
@@ -1,6 +1,10 @@
 parameters:
   argocd:
     namespace: syn
+    monitoring:
+      enabled: true
+      dashboards: false
+      prometheus: platform
```

Comment:
Not sure if adding _instance_ and or _name_ helps to grasp what this parameters is all about.